src/coreclr/inc/cordebug.idl

```diff
@@ -6805,18 +6805,18 @@ interface ICorDebugArrayValue : ICorDebugHeapValue
                            length_is(cdim)] ULONG32 dims[]);
 
     /*
-     * HasBaseIndicies returns whether or not the array has base indices.
+     * HasBaseIndices returns whether or not the array has base indices.
      * If the answer is no, then all dimensions have a base index of 0.
      */
 
-    HRESULT HasBaseIndicies([out] BOOL *pbHasBaseIndicies);
+    HRESULT HasBaseIndices([out] BOOL *pbHasBaseIndices);
```

Comment:
This is re-introducing the breaking change from https://github.com/dotnet/runtime/issues/76960